specification/purview/data-plane/Azure.Analytics.Purview.SelfServicePolicies/preview/2022-12-01-preview/purviewSelfServicePolicy.json

```diff
@@ -0,0 +1,348 @@
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "SelfServicePolicy_Delete": {
+            "$ref": "./examples/SelfServicePolicy_Delete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SelfServicePolicyDecisionRule": {
+      "type": "object",
+      "description": "The decision rule of a SelfServicePolicy",
+      "required": [
+        "effect",
+        "purviewRoleName",
+        "resourceAzurePath"
+      ],
+      "properties": {
+        "effect": {
+          "description": "The effect for rule",
+          "enum": [
+            "Permit"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "Decision",
+            "modelAsString": true
+          }
+        },
+        "purviewRoleName": {
+          "type": "string",
+          "description": "This is field will indicate the role of a self service policy."
+        },
+        "principal": {
+          "type": "string",
+          "description": "Azure Active Directory Id for users"
+        },
+        "principalGroup": {
+          "type": "string",
+          "description": "Azure Active Directory Id for groups"
+        },
+        "resourceAzurePath": {
+          "type": "string",
+          "description": "The resource path of the azure resource"
+        }
+      }
+    },
+    "ErrorModel": {
+      "description": "The error model for policy",
+      "required": [
+        "code",
+        "message"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The error code",
+          "minLength": 1,
+          "type": "string"
+        },
+        "message": {
+          "description": "The error message",
+          "minLength": 1,
+          "type": "string"
+        },
+        "details": {
+          "description": "The error details",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorModel"
+          }
+        }
+      }
+    },
+    "ErrorResponseModel": {
+      "description": "The error response model for policy",
+      "required": [
+        "error"
+      ],
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorModel"
+        }
+      }
+    },
+    "SelfServicePolicy": {
+      "description": "The Self-service policy model",
+      "required": [
+        "decisionRule",
+        "workflowRunId"
+      ],
+      "type": "object",
+      "properties": {
+        "workflowRunId": {
+          "type": "string",
+          "description": "The workflowRunId of the workflow which created the policy"
+        },
+        "requestor": {
+          "type": "string",
+          "description": "The requestor of the self-service policy"
+        },
+        "expiryDate": {
```

Comment:
It is good practice to have a naming convention for date-time properties (we are working an update to the Azure REST API guidelines for this). This file has 3 date-time properties and the other two end with "At", so perhaps this property could be named "expiresAt".